daisy_workflows/kernel_commit_query/debian.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -xeu
+
+apt-get update -y && apt-get upgrade -y && apt-get install -y git linux-source
+
+rm -rf /files/distro_kernel && mkdir -p /files/distro_kernel
+tar --strip-components=1 -xvf $(ls /usr/src/linux-source*.tar.xz) \
+    -C /files/distro_kernel
+
+pushd /files/distro_kernel
+rm -rf {.git,.gitattributes,.gitignore}
+popd
+
+apt-get changelog linux-image-cloud-amd64 > /files/kernel.changelog
```

Comment:
IIRC, we're using the backports kernel for the ARM64 images- may also need to make a config for that.
https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy_workflows/image_build/debian/build_fai.py#L108

Reply:
ow yeah! good catch, I should have made it generic enough to assume the installed kernel. Will change. Thanks.